eos-tech-support/eos-download-image

```diff
@@ -25,7 +25,11 @@ def log(*args):
 
 
 def check_call(*args):
-    log('$ ' + ' '.join(map(shlex.quote, args)))
+    if shlex.quote:
```

Comment:
Wait, does this work? Won't this just throw `AttributeError` if it doesn't exist? I think you want `hasattr(shlex, 'quote')`.